build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts

```diff
@@ -89,21 +89,21 @@ const getCodeCoverageSummaryForPackages = (coverageReport: CodeCoverageCompariso
 
 	return `<details><summary><b>${getColorGlyph(coverageReport.branchCoverageDiff + coverageReport.lineCoverageDiff)} ${
 		coverageReport.packagePath
-	}:</b> <br> Line Coverage Change: ${formatDiff(coverageReport.lineCoverageDiff)}  Branch Coverage Change: ${formatDiff(
+	}:</b> <br> Line Coverage Change: ${formatDiff(coverageReport.lineCoverageDiff)}&nbsp;&nbsp;&nbsp;&nbsp;Branch Coverage Change: ${formatDiff(
 		coverageReport.branchCoverageDiff,
 	)}</summary>${metrics}</table></details>`;
 };
 
 const getColorGlyph = (codeCoverageDiff: number): string => {
 	if (codeCoverageDiff === 0) {
-		return "■";
+		return "&rarr;";
 	}
 
 	if (codeCoverageDiff > 0) {
-		return "⯅";
+		return "&uarr;";
 	}
 
-	return "⯅⯅";
+	return "&darr;&darr;";
```

Comment:
Single glyph for all cases? I don't see an advantage to putting two identical ones in this case. Also curious, have we seen this rendered? The fact that they show up next to the arrow for expand/collapse the details (see at the bottom) makes me want to avoid the arrows altogether. Or since this is HTML, if we style them so the negative case shows red arrows and the others show green ones, that would help.
```suggestion
	return "&darr;";
```

![image](https://github.com/user-attachments/assets/6823b60b-8a45-41c4-b9df-ab78610e095e)

Reply:
Markdown does not support HTML coloring. So, that is why I removed it. Double down arrow is just to draw attention to the regressed package. Here is an example which shows the current change:
https://github.com/microsoft/FluidFramework/pull/22762#pullrequestreview-2355734683

Reply:
Oh, those arrows (at least as I see them! 😄 ) are very distinct so we're good then. For reference, this is what I see:

![image](https://github.com/user-attachments/assets/3aff474d-2628-4ba0-a81a-5a9da5e34ec8)

I still don't see the advantage of the double arrow tbh, but not a blocker.